main/ui/src/main/java/org/cryptomator/ui/common/VaultService.java

```diff
@@ -24,22 +25,29 @@
 
 	private final ExecutorService executorService;
 
+	private AtomicReference<Volume.Revealer> vaultRevealer;
+
```

Comment:
remove this, just pass the revealer to `createRevealTask (...)`

Reply:
Resolved in d5eb84a